sros2/test/sros2/commands/security/verbs/test_list_enclaves.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Canonical Ltd
```

Comment:
code copied without history ?

Reply:
Oh come _on_ github. I need to relearn the trick here...

Reply:
Sadly I can't get this to work. I tried just committing the rename before modification, but [no luck](https://github.com/ros2/sros2/compare/master...kyrofa:test). It seems --follow is the only way. I [found this](https://github.community/t/renaming-folder-within-a-repo-loses-file-history/1752) which seems to confirm there's no way to help git track renames like this. I'm a bit surprised what with the horrible hacks we were able to pull off when reorganizing the API in the first place, I don't know why a straight rename is so complicated. Any ideas?